src/test/java/org/apache/commons/mail/EmailTest.java

```diff
@@ -243,22 +246,25 @@ public void testGetSetSmtpPort()
                 Integer.parseInt(email.getSmtpPort()));
     }
 
-    @Test(expected = IllegalArgumentException.class)
+    @Test
     public void testSetSmtpPortZero()
     {
-        email.setSmtpPort(0);
+        final Executable testMethod = () -> email.setSmtpPort(0);
+        assertThrows(IllegalArgumentException.class, testMethod);
```

Comment:
Hello @nhojpatrick 
Thank you for your PR.
As with the other PRs, the local variable is superfluous. 
Please rebase on master to fix the spotbugs build on Java 11 and 17.